yk_utils/images/__init__.py

```diff
@@ -0,0 +1,2 @@
+""" Image Module Init """
+from . import image_operations
```

Comment:
We can keep the previous file name `image_parser.py`.
We also want to import the functions directly (as it was before):
`from .image_parser import parse_image`